lustre_exporter_test.go

```diff
@@ -0,0 +1,1362 @@
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "3"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "4"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "5"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "6"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "7"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "8"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "read"}, {"ost", "lustrefs-OST0000"}, {"size", "9"}}, 0, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "1"}}, 3.9683886e+07, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "10"}}, 10445, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "11"}}, 1172, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "12"}}, 254, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "13"}}, 47, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "14"}}, 10, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "2"}}, 4.204676e+06, false},
+		{"lustre_disk_io", "Current number of I/O operations that are processing during the snapshot.", gauge, []labelPair{{"operation", "write"}, {"ost", "lustrefs-OST0000"}, {"size", "3"}}, 7
```

Comment:
Did you manually create all these, or did you write code that generated them in some way? Mostly curious.